Package.swift

```diff
@@ -0,0 +1,16 @@
+// swift-tools-version:5.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+import PackageDescription
+
+let package = Package(
+    name: "RxWebKit",
+    products: [
+        .library(name: "RxWebKit", targets: ["RxWebKit"])
+    ],
+    dependencies: [
+        .package(url: "https://github.com/ReactiveX/RxSwift", from: "4.5.0")
+    ],
+    targets: [
+        .target(name: "RxWebKit", dependencies: ["RxSwift", "RxCocoa"], path: "RxWebKit")
```

Comment:
RxWebKit has tests - the Package.swift file should have a test target as well

Reply:
I'm not sure I understand, tests targets should not be exposed in the SPM manifest file should they?

Reply:
AFAIK Package.swift targets are used for public targets but test targets are only for running “Swift test” etc.